src/components/Tooltip/TooltipArrow.css

```diff
@@ -0,0 +1,15 @@
+.arcade-tooltip[data-popper-placement^="top"] > .arcade-tooltip-arrow {
```

Comment:
Please move custom styles into `src/styles.css` and maybe add a `tw-` prefix to keep it consistent